tests/DYCOMS2D/gnuplot.hpp

```diff
@@ -30,27 +30,27 @@ void init_prof(
 }
 
 void init(
-  Gnuplot &gp, 
-  const std::string &file, 
-  const int &ny, const int &nx, 
+  Gnuplot &gp,
+  const std::string &file,
+  const int &ny, const int &nx,
   std::map<std::string, double> n
 )
 {
   boost::filesystem::create_directories(
     boost::filesystem::path(file).parent_path()
   );
 
-  gp << "set term svg dynamic enhanced fsize 13 size " << nx * 500 << "," << ny * 500 << "\n";
+  gp << "set term png size " << nx * 500 << "," << ny * 500 << "\n";
   gp << "set size square\n";
```

Comment:
it's probably better to keep the svg as default

Reply:
Oh yeah for sure. I don't actually remember changing this, but I must have kept the wrong line during some merge! These tests aren't run anyway, so it doesn't really matter. But I'll change it back :)